config/foreman-proxy-content.migrations/230207194222-add-ssh-keepalive-ansible.rb

```diff
@@ -0,0 +1,5 @@
+ansible_module = 'foreman_proxy::plugin::ansible'
+
+if answers[ansible_module].is_a?(Hash) && answers[ansible_module]['ssh_args'] == '-o ProxyCommand=none -C -o ControlMaster=auto -o ControlPersist=60s'
+  answers[ansible_module]['ssh_args'] = '-o ProxyCommand=none -C -o ControlMaster=auto -o ControlPersist=60s -o ServerAliveInterval=15 -o ServerAliveCountMax=3'
```

Comment:
Perhaps this is more expressive:
```suggestion
  answers[ansible_module]['ssh_args'] += ' -o ControlPersist=60s -o ServerAliveInterval=15 -o ServerAliveCountMax=3'
```